### PR TITLE
Add html

### PR DESCRIPTION
**Pull request by Kimutai01**
-Added Html file
-Added h1 tag
-Added css file
-Styled h1 tag

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+# HTML & CSS3 Course
+
+If you are not familiar with linters, read [root level README](../README.md).
+
+## Set-up GitHub Actions
+
+Please do the following **steps in this order**:
+
+1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
+   - **Remember** to use the file linked above
```
Comment (elmar8287) [inline README.md]: - [x] Please, create a detailed README file. Use [MICROVERSE template](https://github.com/microverseinc/readme-template), as required.

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @Kimutai01,

Great work on making the project requirements 👏 

There are still some requirements that aren't done yet, please work on it and ask for another review.
      
 ## Required Changes ♻️

- [x] Please, consider comments under review
- [x] Please, consider the requirement of the project: `Add 3 files (each one in a separate commit)`. You added `.html` and `.css` files, but `test.md` file is missing (do not forget to add the text `This file should be ignored by git` as required in project)
- [x] Please, add `.gitignore` file with required file inside (test.md should be ignored).

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (Kimutai01) [reply README.md]: I have already made the changes

**Review by elmar8287 — APPROVED** [review]
Hi @Kimutai01  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉🎉🎉 

Cheers and Happy coding!👏👏👏

- My suggestion: 

- [ ] Please, change this screenshot in your README file with a screenshot of your project

Screen Shot 2022-01-05 at 23 45 51


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**